label-pull-requests/main.js

```diff
@@ -74,6 +74,7 @@ async function main() {
                 if (
                     (!constraint.status || file.status == constraint.status) &&
                     (!constraint.path || file.filename.match(constraint.path)) &&
+                    (!constraint.except || (Array.isArray(constraint.except) ? constraint.except.includes(file.filename) : file.filename == constraint.except)) &&
```

Comment:
Might be nice to split this onto multiple lines or a separate variable.

Reply:
What do you think about this?

```js
(!constraint.except ||
    (Array.isArray(constraint.except) ?
        constraint.except.includes(file.filename) :
        file.filename == constraint.except)) &&
```

Reply:
Looks better! Think ternary would be nice to avoid here but don't feel strongly.